client/src/css/style.css

```diff
@@ -0,0 +1,100 @@
+.bedroomNum{
+  word-wrap: break-word !important;
```

Comment:
Is there a way that you can avoid using !important? It's in general better practice to not rely on !important